doc/source/dev/governance/governance.rst

```diff
@@ -378,6 +378,22 @@ A list of current Institutional Partners is maintained at the page
 :ref:`governance-people`.
 
 
+Commit Rights in the NumPy Code Base
+====================================
+
+The NumPy Steering Council can grant and revoke commit rights to the NumPy code
+repository, currently hosted at `github`_.
+
+The policy for granting commit priveleges is based on trust, with the
+understanding that they are needed for many of the mundane tasks required
+to maintain NumPy.  Upon receiving this priviledge, it is expected that the
+new committers, until they are experienced enough, will use it conservatively
+to triage issues, add labels, and merge only safe pull requests such as
+documentation.  Any significant code changes should be reviewed by at least
+two experienced NumPy contributors before merging.
```

Comment:
this is not right, we only require one reviewer

Reply:
Hmm, I was told two reviewers are preferred, not required.

Reply:
That's news to me. Agreed that it's nicer to have two reviewers, but given our big backlog of PRs I don't think adding to the review load like that is realistic.

Reply:
I think this is a reasonable requirement for *significant* code changes? One pair of eyes can very easily miss something finicky.  But, perhaps the language can be changed to ensure it is read as a suggestion to be applied specifically for more complex changes?

Reply:
removed controversial sentence

Reply:
I wish more stuff would get merged by single reviewers, we don't have enough reviewers to go around.